packages/flutter_tools/test/integration.shard/analyze_size_test.dart

```diff
@@ -9,14 +9,15 @@ import 'package:file_testing/file_testing.dart';
 import 'package:flutter_tools/src/base/io.dart';
 
 import '../src/common.dart';
+import '../src/context.dart';
 import 'test_utils.dart';
 
 const String apkDebugMessage = 'A summary of your APK analysis can be found at: ';
 const String iosDebugMessage = 'A summary of your iOS bundle analysis can be found at: ';
 const String runDevToolsMessage = 'flutter pub global activate devtools; flutter pub global run devtools ';
 
 void main() {
-  testWithoutContext('--analyze-size flag produces expected output on hello_world for Android', () async {
+  testUsingContext('--analyze-size flag produces expected output on hello_world for Android', () async {
```

Comment:
This didn't even pass locally on master, it needs the context shutdown hook.

Reply:
This is definitely running on CI right? How would it be passing otherwise

Reply:
It passes on CI but not locally because the code that uses the context is in `getFlutterRoot`, which gets bypassed on LUCI because `FLUTTER_ROOT` is set.  I explained it a bit in https://github.com/flutter/flutter/pull/79975.

Really I need to fix the underlying problem, but I've prioritized patching up these integration tests with `testUsingContext` so I can at least run them locally.